utils/scripts/testconfig/test-variables.yaml

```diff
@@ -21,3 +25,7 @@ RDS:
   AWS_ACCESS_KEY_ID: xxxx
   AWS_SECRET_ACCESS_KEY: xxxx
   AWS_REGION: xxxx
+DBAASPOLICY:
+  ENABLED: FALSE
+  PERSONA: cluster-adm
+  NAMESPACE: redhat-dbaas-operator
```

Comment:
@harshkumarRH Shouldn't we change this to 'operator_namespace' as well like in all your files in this PR?